paper-tabs.html

```diff
@@ -103,6 +103,12 @@
       -webkit-tap-highlight-color: rgba(0,0,0,0);
 
       @apply(--paper-tabs);
+
+      display: flex;
+    }
+
+    :host-context([dir=rtl]) {
+      flex-direction: row-reverse;
```

Comment:
@notwaldorf Thanks, I was wondering where `display: flex;` was being applied! I figured I'd explicitly set it since I wasn't sure if it was just somewhere hard to find in the demo CSS, but I'll take it back out (and add the other mixin).